pkg/volume/iscsi/iscsi_util.go

```diff
@@ -86,10 +91,29 @@ func updateISCSIDiscoverydb(b iscsiDiskMounter, tp string) error {
 			}
 		}
 	}
+
 	return nil
 }
 
+//updateISCSINodeTimeouts will update the timeout settings with the
+//values provided in ISCSIVolumeSource and ISCSIPersistentVolumeSource spec
+func updateISCSINodeTimeouts(b iscsiDiskMounter, tp string) {
+	//Updating the iSCSI timeouts settings to keep volume in RW state in the node
```

Comment:
I think you can remove the comments and add them to documentation of iSCSI driver. It's intended to user and not Kubernetes developer, right?